internal/rsl/rsl_test.go

```diff
@@ -43,6 +43,16 @@ func TestInitializeNamespace(t *testing.T) {
 			t.Fatal(err)
 		}
 
+		// Check if RSL with zero hash is treated as uninitialized
+		if err := InitializeNamespace(repo); err != nil {
+			t.Fatal(err)
+		}
+
+		if err := NewReferenceEntry("main", plumbing.ZeroHash).Commit(repo, false); err != nil {
```

Comment:
```suggestion
		if err := NewReferenceEntry("refs/heads/main", plumbing.ZeroHash).Commit(repo, false); err != nil {
```

I'm guilty of using `main` elsewhere in tests but I think it's better to converge on the full path to mimic actual RSL contents. We don't have to go back and fix the other instances right away but if / when we modify those lines, we can take care of it.